conf/app.routes

```diff
@@ -293,3 +293,10 @@ GET        /:taxYear/enhanced-structure-building-allowance/claim-amount
 POST       /:taxYear/enhanced-structure-building-allowance/claim-amount             controllers.EsbaClaimAmountController.onSubmit(taxYear: Int, mode: Mode = NormalMode)
 GET        /:taxYear/enhanced-structure-building-allowance/change-claim-amount      controllers.EsbaClaimAmountController.onPageLoad(taxYear: Int, mode: Mode = CheckMode)
 POST       /:taxYear/enhanced-structure-building-allowance/change-claim-amount      controllers.EsbaClaimAmountController.onSubmit(taxYear: Int, mode: Mode = CheckMode)
+
+GET        /:taxYear/fhl-intro                                                      controllers.furnishedholidaylettings.FhlIntroController.onPageLoad(taxYear: Int)
```

Comment:
Let's ensure uniformity in the URLs. So other pages can follow 
`/furnished-holiday-lettings/start `

Reply:
done. thanks